client/src/main/java/com/vesoft/nebula/client/graph/net/Connection.java

```diff
@@ -0,0 +1,116 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.client.graph.net;
+
+
+import com.facebook.thrift.TException;
+import com.facebook.thrift.protocol.TCompactProtocol;
+import com.facebook.thrift.protocol.TProtocol;
+import com.facebook.thrift.transport.TSocket;
+import com.facebook.thrift.transport.TTransport;
+import com.facebook.thrift.transport.TTransportException;
+import com.vesoft.nebula.client.graph.data.HostAddress;
+import com.vesoft.nebula.client.graph.exception.AuthFailedException;
+import com.vesoft.nebula.client.graph.exception.IOErrorException;
+import com.vesoft.nebula.graph.AuthResponse;
+import com.vesoft.nebula.graph.ErrorCode;
+import com.vesoft.nebula.graph.ExecutionResponse;
+import com.vesoft.nebula.graph.GraphService;
+
+public class Connection {
```

Comment:
Is it similar with GraphClient ?

Reply:
No, it just the connection, the GraphClient can use more than one connection if the connection has broken.